### PR TITLE
Fix Search Tools not working in IE8

### PR DESCRIPTION
**Pull request by C-Lodder**
Pull Request for Issue #11790 
### Summary of Changes

This PR adds a small polyfill for `indexOf()` which is not supported in IE8. Without this, the Search Tools will not work due to Javascript errors.
### Testing Instructions
- Open IE8 (or open IE11 and set it to IE8 emulation)
- Go to Module Manager
- Click on the Search Tools button
### Documentation Changes Required

None

### CODE REVIEW

File: media/jui/js/cms-uncompressed.js [hunk 1]
```diff
@@ -17,6 +17,34 @@ Joomla.setcollapse = function(url, name, height) {
     }
 }
 
+/**
+ * IE8 polyfill for indexOf()
+ */
+if (!Array.prototype.indexOf)
+{
+	Array.prototype.indexOf = function(elt /*, from*/)
```
Comment (zero-24) [inline media/jui/js/cms-uncompressed.js]: just to be sure I'm not good in JS :smile: is the comment needed here?

**Comment by C-Lodder** [comment]
@zero-24 yes, so anyone who looks there knows what it does. It's only for the uncompressed version

**Comment by killoltailored** [comment]
I have tested this item :white_check_mark: successfully on 201cc1dc815d11561976d56e541f79166de7a653<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11794">issues.joomla.org/joomla-cms/11794</a>.</sub>

**Comment by killoltailored** [comment]
Tested successfully with Joomla Version : Joomla! 3.6.3-dev Development [ Noether ] 5-August-2016 00:32 GMT 
And System Windows 8.1, screen shot 2016-08-26 at 03 44 50IE8 emulation<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11794">issues.joomla.org/joomla-cms/11794</a>.</sub>

**Comment by RonakParmar** [comment]
I have tested this item :white_check_mark: successfully on 201cc1dc815d11561976d56e541f79166de7a653<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11794">issues.joomla.org/joomla-cms/11794</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/11794">issues.joomla.org/joomla-cms/11794</a>.</sub>

**Pull request merged**